.junie/skills/pennant-development/SKILL.md

```
@@ -0,0 +1,77 @@
+Feature::for($user)->activate('new-dashboard');
+```
+
+## Verification
+
+1. Check feature flag is defined
+2. Test with different scopes/users
+
+## Common Pitfalls
+
+- Forgetting to scope features for specific users/entities
+- Not following existing naming conventions
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)